rules.d/70-uaccess.rules.in

```diff
@@ -99,4 +99,8 @@ SUBSYSTEM=="hidraw", ENV{ID_AV_PRODUCTION_CONTROLLER}=="1", TAG+="uaccess"
 # This also allows accessing HID devices with the libusb backend of hidapi.
 SUBSYSTEM=="usb", ENV{ID_AV_PRODUCTION_CONTROLLER}=="1", TAG+="uaccess"
 
+# Hardware wallets
+SUBSYSTEM=="usb", ENV{ID_HARDWARE_WALLET}=="1", TAG+="uaccess"
+KERNEL=="hidraw*", ENV{ID_HARDWARE_WALLET}=="1", TAG+="uaccess"
```

Comment:
SUBSYSTEM=="hidraw"?

Reply:
I tested it with `SUBSYSTEM=="hidraw*"` and it seems to be working, so I'll change it to that then.
Also rebased the code and squashed the commits.